src/api.js

```diff
@@ -0,0 +1,15 @@
+var express = require('express');
+var app = express();
+
+app.get('/stub', function(req, res) {
+    res.redirect('http://31.media.tumblr.com/65f2c221257116959599522008abfa4c/tumblr_n7qalc0aPr1r7tu05o2_500.gif');

```

Comment:
1. Should we have add some error handling here in case the resource (aka "gif") isnt available?